### PR TITLE
Replace goja by otto which support Default JS function like console.log

### PR DESCRIPTION
**Pull request by weburnit**
I've changed some function to private, because those are no need to expose outside package

### CODE REVIEW

**Comment by flogo-oss** [comment]
[CLA assistant check](https://flogo.cla.cloud.tibco.com:443/project-flogo/contrib?pullRequest=111) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://flogo.cla.cloud.tibco.com:443/project-flogo/contrib?pullRequest=111) before we can accept your contribution.<br/><hr/>**Paul Aan** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://flogo.cla.cloud.tibco.com:443/check/project-flogo/contrib?pullRequest=111) it.</sub>

**Comment by skothari-tibco** [comment]
Any particular reason to switching to otto? goja seems to be more active than otto.

**Comment by weburnit** [comment]
> Any particular reason to switching to otto? goja seems to be more active than otto.

goja doesn't support default function like `console.log()` I've tested and make this PR to fix that issue

File: activity/jsexec/activity.go [hunk 1]
```diff
@@ -52,30 +52,29 @@ func (a *Activity) Eval(ctx activity.Context) (done bool, err error) {
...
 	}
-	err = vm.GetFromVM("result", &result)
+	result, err := vm.getFromVM("result")
 	if err != nil {
 		output.Error = true
 		output.ErrorMessage = err.Error()
 		return false, err
 	}
-	output.Result = result
+	output.Result = result.(map[string]interface{})
```
Comment (skothari-tibco) [inline activity/jsexec/activity.go]: Can you change it to `output.Result, err = coerce.ToMap(result)`. This way it minimizes the risk of run-time error.

**Comment by fm-tibco** [comment]
Goja is significantly faster than Otto.  If console is the only issue, it can be added to goja using the following:
```
import (
        ...
        "github.com/dop251/goja_nodejs/console"
        "github.com/dop251/goja_nodejs/require"
)

...
        vm := goja.New()
        new(require.Registry).Enable(vm)
        console.Enable(vm)
```

**Comment by weburnit** [comment]
> Goja is significantly faster than Otto. If console is the only issue, it can be added to goja using the following:
> 
> ```
> import (
>         ...
>         "github.com/dop251/goja_nodejs/console"
>         "github.com/dop251/goja_nodejs/require"
> )
> 
> ...
>         vm := goja.New()
>         new(require.Registry).Enable(vm)
>         console.Enable(vm)
> ```

if its faster than otto, we can get rid of otto

**Comment by fm-tibco** [comment]
@weburnit You can create a PR with what I suggested above to add console.log() support

**Comment by weburnit** [comment]
@fm-tibco I figured out goja support ECMAScript 5 which doesn't have `import statement`

**Comment by fm-tibco** [comment]
Yes, import is a part of ECMAScript 6, which goja doesn't support.  I guess we'll have to evaluate if import makes sense for this activity.  We could support require as an alternative to import with goja.  https://github.com/dop251/goja_nodejs

**Pull request closed without merging**